lib/logflare_web/controllers/plugs/set_verify_user.ex

```diff
@@ -42,56 +42,37 @@ defmodule LogflareWeb.Plugs.SetVerifyUser do
         other ->
           other
       end
-      |> case do
-        id when is_integer(id) ->
-          Users.get_by_and_preload(id: id)
-          |> Users.preload_team()
-          |> Users.preload_billing_account()
-          |> Users.preload_sources()
-
-        _ ->
-          nil
-      end
+      |> fetch_preloaded_user_by_id()
 
     assign(conn, :user, user)
   end
 
   defp set_user_for_mgmt_api(conn, _opts) do
```

Comment:
mgmt api auth is handled by VerifyApiAccess, so we can actually remove this chunk of code.